ceph.spec.in

```diff
@@ -537,12 +537,14 @@ Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-jwt
 Requires:       python%{python3_pkgversion}-routes
 Requires:       python%{python3_pkgversion}-werkzeug
+Suggests:       python%{python3_pkgversion}-saml
 %endif
 %if 0%{?suse_version}
 Requires:       python%{python3_pkgversion}-CherryPy
 Requires:       python%{python3_pkgversion}-PyJWT
 Requires:       python%{python3_pkgversion}-Routes
 Requires:       python%{python3_pkgversion}-Werkzeug
+Suggests:       python%{python3_pkgversion}-python3-saml
```

Comment:
after reading the "Files" section in https://opensuse.pkgs.org/tumbleweed/opensuse-oss-aarch64/python2-python3-saml-1.9.0-1.1.noarch.rpm.html , i am convinced that this python2-python3-saml does not offer python3-saml.

Reply:
This python3-saml package is really weird one, as it provides both python2- and python3-, but then the same author provides a separate python-saml that only works with Python 2.7... Nevertheless, I removed that weak dep and left only Fedora/CentOS'.